src/components/UI/Buttons/CircleCloseButton/CircleCloseButton.tsx

```diff
@@ -0,0 +1,17 @@
+'use client'
+import Image from 'next/image'
+import CloseIcon from '@/../public/close.svg'
+
+export default function CircleCloseBtn({ onClick }: { onClick: () => void }) {
+  return (
+    <div onClick={onClick} onKeyDown={onClick}>
+      <button
```

Comment:
QA engineers asked us to ad some kind of identifiers to the interactive elements, including buttons. We agreed on adding id.  So it could be something like this :  <button id="close-btn"> Close</buttton>

Reply:
Fixed